catalogs/hadoop-compatible-common/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+plugins {
+  id("java")
+}
+
+group = "org.example"
+version = "0.6.0-SNAPSHOT"
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  implementation(project(":common"))
+  implementation(project(":core"))
+  implementation(libs.bundles.log4j)
+  implementation(libs.slf4j.api)
+
+  implementation(libs.guava)
+  implementation(libs.hadoop3.common) {
+    exclude("com.sun.jersey")
+    exclude("org.apache.hadoop.hdfs.web")
+    exclude("javax.servlet", "servlet-api")
+    exclude("org.eclipse.jetty")
+  }
+
+  implementation(libs.hadoop3.hdfs) {
+    exclude("com.sun.jersey")
+    exclude("org.apache.hadoop.hdfs.web")
+    exclude("javax.servlet", "servlet-api")
+    exclude("org.eclipse.jetty")
+  }
+  implementation(libs.hadoop3.client) {
+    exclude("com.sun.jersey")
+    exclude("org.apache.hadoop.hdfs.web")
+    exclude("javax.servlet", "servlet-api")
+    exclude("org.eclipse.jetty")
+  }
```

Comment:
I think Hive catalog is using hadoop2, what will be happened if you change to hadoop3 in this module and make hive catalog dependable on this module?

Reply:
So, we need to implement a separate logic for Kerberos authentication in Hive and Hadoop catalogs?